lib/index.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
   rules: {
+    'a11y-no-sr-only-class-when-focusable': require('./rules/a11y-no-sr-only-class-when-focusable'),
```

Comment:
Should we wait to include this until: https://github.com/github/accessibility/issues/3868

Reply:
I think it's fine! We could always override disable is temporarily in our projects.